3D-microservices-observability/springboot-jersey-app-2/README.md

```diff
@@ -0,0 +1,78 @@
+# Wavefront Distributed Tracing Sample Apps - Springboot Jersey App 2
+
+This is a sample java application using Springboot beachshirts (#[beachops](https://medium.com/@matthewzeier/thoughts-from-an-operations-wrangler-how-we-use-alerts-to-monitor-wavefront-71329c5e57a8)) 
+which makes shirts for the beach. The application is based on spring boot web using rest templates and rest annotations.
+- JDK 1.8 and up
+- Spring Boot 2.3.0.RC1
+
+## Running Application locally 
+
+- `git clone` this repo and navigate to this dir:
+
+- ```bash
+  git clone https://github.com/howardyoo/distributed-tracing-sample-apps/
+  cd ./distributed-tracing-sample-apps/3D-microservices-observability/springboot-jersey-app-2
+  ```
+- Run `mvn clean compile` from the root directory of the project.
+
+- Now run all the services using below commands from root directory of the project.
+
+  ```bash
+  mvn -pl registration spring-boot:run &
+  mvn -pl delivery spring-boot:run &
```

Comment:
Running the `registration` app works with no errors. However, this and the remaining two apps fail with:

```
[INFO]
[INFO] -----------------------< com.wavefront:delivery >-----------------------
[INFO] Building delivery 0.0.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] >>> spring-boot-maven-plugin:2.3.0.RC1:run (default-cli) > test-compile @ delivery >>>
[WARNING] The POM for com.wavefront:common:jar:0.0.1-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.006 s
[INFO] Finished at: 2021-08-30T09:10:08-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project delivery: Could not resolve dependencies for project com.wavefront:delivery:jar:0.0.1-SNAPSHOT: com.wavefront:common:jar:0.0.1-SNAPSHOT was not found in https://repo.spring.io/milestone during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of spring-milestones has elapsed or updates are forced -> [Help 1]
[ERROR]
```